product_cost_security/views/product_views.xml

```diff
@@ -6,21 +6,22 @@
         <field name="arch" type="xml">
             <xpath expr="//div[@name='standard_price_uom']" position="inside">
                 <field name="user_can_update_cost" invisible="1" />
+                <field name="user_can_see_standard_price_uom" invisible="1" />
             </xpath>
             <xpath expr="//div[@name='standard_price_uom']" position="attributes">
                 <attribute
-                    name="groups"
-                    separator=","
-                    add="product_cost_security.group_product_cost"
```

Comment:
In the V15 version, group `product_cost_security.group_product_cost` was added to already existing "`base.group_user`"  for this field. So in theory you may have multiple groups for `standard_price_uom`.
But in `_compute_user_can_see_standard_price_uom()` you only handle one group. Why did you add `product_variant_count` and `is_product_variant` in the condition? Can't you just leave things as they were in V15?

Reply:
Sorry for the inconveniences caused, this happened as the Odoo base addons were recently changed(https://github.com/odoo/odoo/commit/206e88b0ad9fdb6e1767c93b3eca8feeecddd3da) and I had an old db where the group was still applied on the field and it was causing an additional field (uom) to show up, please check attached screenshot where in the view is disturbed with per Units showing up which should have hidden. I have restored it to how it was in v15
![Screenshot from 2023-01-03 16-50-57](https://user-images.githubusercontent.com/88703470/210347866-f9622b41-9bec-4215-82aa-a8a506600e4b.png)